ccloader/data/definitions.db

```diff
@@ -926,6 +940,7 @@
                             "type": "member",
                             "refType": "raw",
                             "name": "jump",
+                            "reobfName": "doJump",
```

Comment:
...reobfName is the name used for the reobfuscation mapping, which has to be accurate. Changing 'name' for this would cause complaints about compatibility, so I instead opted for this solution.